CHANGELOG.md

```diff
@@ -22,6 +22,9 @@ _ `_optional` subpackage for managing optional dependencies
 - Abstract `ContinuousNonlinearConstraint` class
 - `ContinuousCardinalityConstraint` class and corresponding uniform sampling mechanism
 - `register_hooks` utility enabling user-defined augmentation of arbitrary callables
+- Abstract `CardinalityConstraint` class
```

Comment:
Just a quick question: Is this PR already ready for review by @Scienfitz and me?

Reply:
This PR is ready for review. Not sure whether @AdrianSosic would like to have a first round.

Reply:
will have now, thus closing this tread 🙃